README.md

```diff
@@ -120,155 +128,221 @@ Client(address='', container='', user='', password='', token='')
+
+```python
+import v3io_frames as v3f
+client = v3f.Client("framesd:8081", user="iguazio", token="e8bd4ca2-537b-4175-bf01-8c74963e90bf", container="users")
+```
 
 ```python
 import v3io_frames as v3f
 client = v3f.Client("framesd:8081", user="iguazio", password="mypass", container="users")
 ```
 
 <a id="client-common-method-params"></a>
-### Common Client Method Parameters
+### Common `Client` Method Parameters
 
-All client methods receive the following common parameters:
+All client methods receive the following common parameters; additional, method-specific parameters are described for each method.
 
 - <a id="client-method-param-backend"></a>**backend** &mdash; The backend data type for the operation.
-  See the backend-types descriptions in the [overview](#backend-types).
+  See [Backend Types](#backend-types).
 
   - **Type:** `str`
-  - **Valid Values:** `"csv"` | `"kv"` | `"stream"` | `"tsdb"`
   - **Requirement:** Required
+  - **Valid Values:** `"csv"` (for testing) | `"kv"` | `"stream"` | `"tsdb"`
 
-- <a id="client-method-param-table"></a>**table** &mdash; The relative path to the backend data &mdash; A directory in the target platform data container (as configured for the client object) that represents a TSDB or NoSQL table or a data stream.
+- <a id="client-method-param-table"></a>**table** &mdash; The relative path to the backend data &mdash; a directory in the target platform data container (as configured for the client object) that represents a TSDB or NoSQL table or a data stream.
   For example, `"mytable"` or `"examples/tsdb/my_metrics"`.
 
   - **Type:** `str`
   - **Requirement:** Required unless otherwise specified in the method-specific documentation
 
-Additional method-specific parameters are described for each method.
-
 <a id="method-create"></a>
-### create Method
+### `create` Method
 
-Creates a new TSDB table or a stream in a platform data container, according to the specified backend type.
+Creates a new TSDB table or stream in a platform data container, according to the specified backend type.
 
 The `create` method is supported by the `tsdb` and `stream` backends, but not by the `kv` backend, because NoSQL tables in the platform don't need to be created prior to ingestion; when ingesting data into a table that doesn't exist, the table is automatically created.
 
 - [Syntax](#method-create-syntax)
 - [Common parameters](#method-create-common-params)
 - [`tsdb` backend `create` parameters](#method-create-params-tsdb)
 - [`stream` backend `create` parameters](#method-create-params-stream)
+- [Examples](#method-create-examples)
 
 <a id="method-create-syntax"></a>
 #### Syntax
+<!-- [IntInfo] (26.9.19) (sharonl) We omitted `schema=None` because the
+  `schema` parameter is used only with the `csv` testing backend. -->
+
+<a id="method-create-syntax-tsdb"></a>
+##### `tsdb` Backend
```

Comment:
The whole `Syntax` part is a bit confusing, i think there is no need for that there is an overview on the backend specific parameters later + examples that's enough

Reply:
I think that as a user, it's helpful to have a syntax section with all relevant params listed in one location. However, I agree that the backend division here and the passing of some params via a parameter (e.g., `attrs`) makes the output a bit awkward. => What if we just removed the separate `attrs` syntax?